From a5ac8e9ce431217c80553eee3f0dc09a4650dc27 Mon Sep 17 00:00:00 2001
From: Sruthik P <spatibandlla@nvidia.com>
Date: Thu, 3 Apr 2025 18:18:15 +0530
Subject: [PATCH] egl-x11: Add support for tegradisp-drm

This change adds support in the X11/XCB EGL Platform library
for the tegradisp-drm DRM driver meant for Automotive (DRIVE OS)
Tegra platforms.
---
 src/x11/x11-platform.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/x11/x11-platform.c b/src/x11/x11-platform.c
index c1fa162..be78a69 100644
--- a/src/x11/x11-platform.c
+++ b/src/x11/x11-platform.c
@@ -1,5 +1,5 @@
 /*
- * SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
  * SPDX-License-Identifier: Apache-2.0
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
@@ -500,6 +500,7 @@ static EGLDeviceEXT FindDeviceForFD(EplPlatformData *plat, int fd)
                 if (version->name != NULL)
                 {
                     if (strcmp(version->name, "nvidia-drm") == 0
+                            || strcmp(version->name, "tegradisp-drm") == 0
                             || strcmp(version->name, "tegra-udrm") == 0
                             || strcmp(version->name, "tegra") == 0)
                     {
-- 
2.43.0

